acos_client/v30/slb/virtual_port.py

```diff
@@ -72,6 +72,9 @@ def _set(self, virtual_server_name, name, protocol, port,
             }, exclude=['template-persist-source-ip', 'template-persist-cookie'])
         }
 
+        sampling_enable = kwargs.get('sampling_enable', [])
+        self._set_sampling_enable(sampling_enable, params)
```

Comment:
This will set sampling enable to `[]` on updates, even if it's not being supplied. `sampling-enable` will be removed by updates.